src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -2557,17 +2559,51 @@ Expression InExpression() #InExpression :
         [ "(" "+" ")" { result.setOldOracleJoinSyntax(EqualsTo.ORACLE_JOIN_RIGHT); } ]
     )
     [<K_NOT> { result.setNot(true); } ] <K_IN>
-        ( rightExpression = Function()
-            | token=<S_CHAR_LITERAL> { rightExpression = new StringValue(token.image); }
-            | "(" (LOOKAHEAD(3) rightItemsList=SubSelect() | rightItemsList=SimpleExpressionList() ) ")")
+    (
+      // syntactic lookahead for a multi expression list, ie: ((a,b),(c,d))
```

Comment:
I read through the options for lookahead and this seemed pretty nifty: https://javacc.github.io/javacc/tutorials/lookahead.html#syntactic

Reply:
However, if the potential performance hit from this is a concern, I can try to use a `LOOKAHEAD(2, "(" "(")` and see how that works.

Reply:
Or a vanilla plain `LOOKAHEAD(3) multiExpressionList = MultiInExpressions()` also works.